util/secure_http.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 )
 
+// New comment here
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function SecureTransport should be of the form "SecureTransport ..."